nuxeo-core/nuxeo-core-convert/src/test/java/org/nuxeo/ecm/core/convert/tests/TestCache.java

```diff
@@ -94,4 +100,33 @@ public void testCache() throws Exception {
         assertEquals(0, cacheSize1 - cacheSize3);
     }
 
+    @Test
+    public void shouldNotReturnNullFileCacheEntries() throws IOException {
+        File file = FileUtils.getResourceFileFromContext("test-data/hello.doc");
+        assertNotNull(file);
+        assertTrue(file.length() > 0);
```

Comment:
I know these 2 lines are in the existing test, but I wouldn't keep them as they're asserting on a static test resource from the repository.

Reply:
Yes taken off !